content/docs/user-guide/experiment-management/comparing-experiments.md

```diff
@@ -3,6 +3,17 @@
 DVC provides commands to list, tabulate and contrast experiments. Let's see how
 they can help you streamline the experimentation process.
 
+<admon type="tip">
+
+**New!** You can compare your ML experiments with DVC directly [from Visual
+Studio Code].
+
+https://www.youtube.com/watch?v=LHi3SWGD9nc
```

Comment:
Not sure I like the huge video taking up so much space and also, the video is pretty general, not specifically about comparing exps.

Reply:
Good points. It's already in the index/overview, so I'm dropping the video from here.